tests/cnf/ran/internal/ranparam/const.go

```diff
@@ -35,6 +35,10 @@ const (
 	RetryInterval = 10 * time.Second
 	// RetryCount retry count for node exec commands.
 	RetryCount = 3
+	// SNO represents spoke cluster type as single-node openshift (SNO) cluster.
+	SNO = "sno"
+	// Standard represents spoke cluster type as multi-node openshift (MNO) cluster.
+	Standard = "standard"
```

Comment:
Could you make the cluster type into a separate type like what HubOperatorName does?

Reply:
Okay, updated. Thanks!